opensquirrel/passes/merger/single_qubit_gates_merger.py

```diff
@@ -0,0 +1,56 @@
+from typing import cast
+
+from opensquirrel.circuit import Circuit
+from opensquirrel.default_instructions import I
+from opensquirrel.ir import Barrier, BlochSphereRotation, Instruction, Qubit
+from opensquirrel.passes.merger.general_merger import Merger, compose_bloch_sphere_rotations, try_name_anonymous_bloch
+
+
+class SingleQubitGatesMerger(Merger):
+    def merge(self, circuit: Circuit) -> None:
```

Comment:
As said before, there is no point of this method not being static (at least for now).